src/UserProxy.sol

```diff
@@ -78,7 +81,15 @@ contract UserProxy is IUserProxy {
             require(success, "UserProxy: eth-fail");
         }
 
-        emit Unstake(_amount, _recipient, lusdAmount, ethAmount);
+        emit Unstake(
+            _recipient,
+            _amount,
```

Comment:
In my opinion `_amount` is meaningless, and we should calculate the balance difference like we do for LUSD & ETH and use that instead. This is because one you pass any amount higher than your stake, and `LQTYStaking` will internally truncate it to the maximum withdrawable amount.